src/controller/src/clusters.rs

```diff
@@ -425,6 +436,12 @@ where
                             name: "storagectl".into(),
                             port_hint: 2100,
                         },
+                        // To simplify the changes to tests, the port
+                        // chosen here is _after_ the compute ones.
```

Comment:
Can you add a `TODO(petrosagg)` to fix it? It bothers be enough that I'm willing to do it after this gets merged

Reply:
thanks for offering!